README.md

```diff
@@ -1,17 +1,21 @@
 # PYMMH3 - a pure python MurmurHash3 implementation.
 
-[![Build Status](https://travis-ci.org/wc-duck/pymmh3.svg?branch=master)](https://travis-ci.org/wc-duck/pymmh3)
+[![Build Status](https://travis-ci.org/swapniel99/pymmh3.svg?branch=master)](https://travis-ci.org/swapniel99/pymmh3)
 
 ## About:
 This is a pure python implementation of the murmur3 hash algorithm <https://code.google.com/p/smhasher/wiki/MurmurHash3>
 written for the times when you do not want to muck about building and installing c-modules etc.
 
-As this is pure python performance is FAR from optimal and if that is required a real c-module such as the
-one found here at <https://pypi.python.org/pypi/mmh3/2.0> is absolutely suggested!
+As this is pure python performance is FAR from optimal in CPython and if that is required a real c-module such as the
+one found here at <https://pypi.python.org/pypi/mmh3/2.3.1> is absolutely suggested!
+
+However this module performs extremely well in case of pypy(<http://pypy.org>), an alternative JIT version of python.
```

Comment:
What I ask is, how was the c-module built? I see that the JIT in pypy could work well with this code but I have a hard time to see that there should be x20 overhead in calling into a .pyd (dll), reading i string ( that is basically just passing a pointer ), and returning a 32-bit value.
I might be wrong but for me it sounds like something is off.
Was it you who built mmh3? In that case, how?
I might be wrong, I have been before ;) I just don't want to claim things that might not be true.